argocon-demos/2026-bring-your-own-artifact-driver-to-workflows/README.md

```
@@ -0,0 +1,20 @@
+# Bring Your Own Artifact Driver To Workflows
+
+[![Pipekit Logo](https://raw.githubusercontent.com/pipekit/talk-demos/main/assets/images/pipekit-logo.png)](https://pipekit.io?utm_campaign=talk-demos)
+
+
+<!-- ## The talk
+The talk recording [can be found here]().-->
+
+The slide deck for this talk can be found [here](assets/slide-deck.pdf).
```

markdownlint-cli2 (0.21.0)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)